mobile/lib/main.dart

```diff
@@ -66,6 +66,12 @@ void main() {
   config.customOpeningDivider = "| ";
 
   FLog.applyConfigurations(config);
+
+  // Based on GoRouter migration guide to preserve old behaviour
+  // https://docs.google.com/document/d/1VCuB85D5kYxPR3qYOjVmw8boAGKb7k62heFyfFHTOvw/edit
+  // TODO: Verify whether we need this setting
+  GoRouter.optionURLReflectsImperativeAPIs = true;
```

Comment:
🔧 I tested the app after updating and we didn't need this as far as I could tell.

Reply:
even better; I erred on the side of caution in the beginning :-) 

thank you for double-checking this!